docs/concepts/privacy-and-security.md

```diff
@@ -212,10 +214,18 @@ The following are some examples of how you could get confirmation.
 
 - Send a text message to the user that includes a confirmation code that the user can enter in the add-in.
 
-- Open a confirmation dialog in a new browser window to a page that cannot be iframed. This is typically the pattern that is used by login pages. Use the [dialog api](../develop/dialog-api-in-office-add-ins.md) to create a new dialog.
+- Open a confirmation dialog in a new browser window to a page that cannot be iframed. This is typically the pattern that is used by login pages. Use the [dialog API](../develop/dialog-api-in-office-add-ins.md) to create a new dialog.
 
 Also, ensure that the address you use for contacting the user couldn't have been provided by a threat actor. For example, for payment confirmations use the address on file for the authorized user's account.
 
+### Request permission to access device capabilities (applies to Office on the web)
+
+On Chromium-based browsers (such as Microsoft Edge or Google Chrome), if an add-in running in Office on the web requires access to a user's device capabilities, a developer must configure their add-in to request permission from the user. This applies to web versions of Excel, Outlook, PowerPoint, and Word.
+
+A user's device capabilities include their camera, geolocation, and microphone. To request permission, the add-in must implement the [device permission API](/javascript/api/office/office.devicepermission).
+
+For information on how the user is prompted for permission, see [View, manage, and install add-ins for Excel, PowerPoint, and Word](https://support.microsoft.com/office/16278816-1948-4028-91e5-76dca5380f8d).
+
```

Comment:
Could we add a sentence or two about what happens if you're not on the web? And not on a Chromium browser? Do you just already have access?

Reply:
Added.